build_compiler.js

```diff
@@ -23,7 +25,7 @@ if (compilerJarStats && compilerJarStats.isFile()) {
   for (let line of versionOutput.output) {
     if (line) {
       const lineString = line.toString();
-      const versionParts = /^Version: v(\d+)$/m.exec(lineString);
+      const versionParts = /^Version: v(\d+)(?:[-\.][-a-z0-9]+)?$/m.exec(lineString);
```

Comment:
How did you derive this pattern?  If you change the `?` to be a `*`, you could match semver versions.

Reply:
There are three version patterns on the closure-compiler repo as of right now:

 - v20180610
 - v20180610-gwt-fix
 - v20180610.webpack-beta

The pattern was simply adjusted to match more than the first pattern. I'll change it to '*' though